packages/cursorless-everywhere-talon/src/test/quickjsTest.ts

```diff
@@ -0,0 +1,93 @@
+import { getTestHelpers } from "talon";
```

Comment:
why do you need these? how come you can't just directly call the talon actions from here?

Reply:
I don't want all of this exposed to cursorless. A lot of this stuff should only be in the test runner.